### PR TITLE
Adding a popular bash book named as Linux Shell Scripting with bash By Ken O. Burtch

### PR DESCRIPTION
**Pull request by CyberSecByte**
## What does this PR do?
Add resource

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -396,6 +396,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Introduction to the Command Line](https://launchschool.com/books/command_line) - Launch School (HTML)
 * [Linux Shell Scripting Tutorial - A Beginner's Handbook (2002)](http://www.freeos.com/guides/lsst/) - Vivek G. Gite (HTML)
 * [Linux Shell Scripting Tutorial (LSST) v2.0](https://bash.cyberciti.biz/guide/Main_Page) - Vivek Gite (HTML)
+* [Linux Shell Scripting With Bash](https://archive.org/download/B-001-002-839/LinuxShellScriptingWithBash-Sams.pdf) - Ken O. Burtch (PDF)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:399]: Is this resource free?

Comment (CyberSecByte) [reply books/free-programming-books-langs.md:399]: yes @shrn01 the resource is completely free

Comment (eshellman) [reply books/free-programming-books-langs.md:399]: add the 'archived' notation

Comment (CyberSecByte) [reply books/free-programming-books-langs.md:399]: Oh okay

**Pull request merged**